FileOperationsCases.cs

```diff
@@ -0,0 +1,81 @@
+﻿using HelloWorld.Models;
+using Newtonsoft.Json;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json.Schema;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Text;
+
+namespace HelloWorld
+{
+     public class FileOperationsCases
+    {
+        public void WritingJsonToFile()
+        {
+            HomeActivity homeActivity = new HomeActivity("Ironing", 2, "Saturday");
+
+            string serialized = JsonConvert.SerializeObject(homeActivity, Formatting.Indented);
+
+            if (IsValidJson(serialized))
+                File.WriteAllText(@"C:\Users\obartkiv\source\repos\HelloWorld\FileA.txt", JsonConvert.SerializeObject(homeActivity));
+
+        }
+
+        public void ReadingFromFile()
+        {
+            List<HomeActivity> listOfActivities = new List<HomeActivity>();
+            string line;
+
+            StreamReader file = new StreamReader(@"C:\Users\obartkiv\source\repos\HelloWorld\FileB.txt");
```

Comment:
would also be good checking if file exists before reading e.g. System.IO.File.Exists()